modules/cnv-preparing-to-install.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/installing-container-native-virtualization.adoc
+
+[id="cnv-preparing-to-install_{context}"]
+= Preparing to install {ProductName}
+
+Before deploying {ProductName}, you must create the `kubevirt-hyperconverged`
+namespace. After the namespace is created, a user with `cluster-admin` privileges
+must create a supporting `OperatorGroup` and `CatalogSource`.
+
+.Prerequisites
+
+* {product-title} 4.1
+* User with `cluster-admin` privileges
+
+.Procedure
+
+. Create the `kubevirt-hyperconverged` namespace by executing the following
+command:
++
+----
+$ oc new-project kubevirt-hyperconverged
+----
+
+. Execute the following command to create the `OperatorGroup` and the
+`CatalogSource` for the `kubevirt-hyperconverged` namespace:
++
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: hco-operatorgroup
+  namespace: kubevirt-hyperconverged
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalogsource
+  namespace: openshift-operator-lifecycle-manager
+  imagePullPolicy: Always
+spec:
+  sourceType: grpc
+  image: $PUBLIC_REGISTRY/container-native-virtualization/hco-bundle-registry:v2.0.0-6 <1>
+  displayName: KubeVirt HyperConverged
+  publisher: Red Hat
+EOF
+----
+<1> Replace `$PUBLIC_REGISTRY` with the registry's actual address before
```

Comment:
i think we should directly point here to RH CDN, no?

Reply:
But is it only D/S?
If yes for U/S, then also all images inside CSV (inside catalogsource) should also changed to U/S

Reply:
> i think we should directly point here to RH CDN, no?

Fabian said: "We can and thus should replace the PUBLIC_REGISTRY part, or rather the complete `image:` part once we know it.

Ryan Hallisey and Simone Tiraboschi can provide this once the errata goes into REL_PREP"

Anything I should change before then, @nellyc ?

Reply:
I actually wonder if we already know these two parts (registry and tag) today

Reply:
yes please change to RH prod CDN & just keep tag 2.0.0

Reply:
@nellyc I updated it in this commit: https://github.com/openshift/openshift-docs/pull/14858/commits/bc34188a31bc82ac19c189c528d1ab9a446d5e18